contracts/scripts/governor/propose.js

```diff
@@ -32,7 +31,9 @@ async function proposeVaultv2GovernanceArgs() {
     "AaveStrategy",
     cAaveStrategyProxy.address
   );
-  const cCompoundStrategyProxy = await ethers.getContract("CompoundStrategyProxy");
+  const cCompoundStrategyProxy = await ethers.getContract(
+    "CompoundStrategyProxy"
```

Comment:
There are a lot of changes in this PR that are code style related. Did running `prettier` cause this or your IDE? I would suggest having code style changes in a separate PR (in the future) since it is harder to code review it

Reply:
Okay, I follow this